servers/quarkus-server/src/test/java/org/projectnessie/server/authn/AuthenticationDisabledProfile.java

```diff
@@ -23,7 +23,7 @@
 public class AuthenticationDisabledProfile implements QuarkusTestProfile {
 
   public static final Map<String, String> CONFIG_OVERRIDES =
-      ImmutableMap.of("nessie.auth.enabled", "false");
+      ImmutableMap.of("nessie.authentication.enabled", "false");
```

Comment:
Should this be `nessie.server.authentication.enabled`?

Reply:
ah... should not have renamed late in the evening :facepalm:

Reply:
fixed in a68aa0d0341b44dc7e39c4cf2cfa28f30dcb062e